modules/rhel-compute-requirements.adoc

```diff
@@ -32,12 +32,12 @@ endif::[]
 ** Minimum 8 GB RAM.
 ** Minimum 15 GB hard disk space for the file system containing `/var/`.
 ** Minimum 1 GB hard disk space for the file system containing `/usr/local/bin/`.
-** Minimum 1 GB hard disk space for the file system containing the system's temporary directory. The system’s temporary directory is determined according to the rules defined in the tempfile module in Python’s standard library.
+** Minimum 1 GB hard disk space for the file system containing its temporary directory. The temporary system directory is determined according to the rules defined in the tempfile module in the Python standard library.
 * Each system must meet any additional requirements for your system provider. For example, if you installed your cluster on VMware vSphere, your disks must be configured according to its link:https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/index.html[storage guidelines] and the `disk.enableUUID=true` attribute must be set.
 
-* Each system must be able to access the cluster's API endpoints by using DNS-resolvable host names. Any network security access control that is in place must allow the system access to the cluster's API service endpoints.
+* Each system must be able to access the API endpoints of the cluster by using DNS-resolvable host names. Any network security access control that is in place must allow system access to the API service endpoints of the cluster.
```

Comment:
Would _cluster API endpoints_ and cluster API service endpoints_ work here? A bit tighter. 
Are the API endpoints and API service endpoints different endpoints?

Reply:
Heh, I had the same thought. Was trying to revise per ISG to remove the contraction but in doing so, it only makes it more confusing. Plus, after a bit more digging into the root cause of this wording, I've decided to leave the original version so that I don't mess with the nuances ([example](https://github.com/openshift/openshift-docs/pull/22392#discussion_r481257393)) of what was already reviewed and approved. FWIW, API endpoints and API service endpoints seem to be unique to each sentence, though I couldn't necessarily tell you why. 😁